### PR TITLE
Par 247 implement calculate function

### PR DESCRIPTION
**Pull request by thelostone-mc**
- runs basic calculations
- loads data onto table 
- update fetcher to include OR operators
- include uuid in csv

http://localhost:3000/huss-round/admin/distribute

In the next PR, will tackle
- [ ]  Implement Download All Votes
- [ ] Update table to show % of pool distribution and the actual amount
- [ ] When you import/export, make sure they tweak only % and not amount
- [ ] Fix Create Pool Bug
- [ ] Test Distribute

### CODE REVIEW

File: src/server/api/routers/results.ts [hunk 1]
```diff
@@ -132,32 +134,21 @@ async function calculateBallotResults({
...
-  if (round.type === RoundTypes.impact) {
-    throw new TRPCError({
-      code: "BAD_REQUEST",
-      message: "Calculations for impact rounds not implemented yet",
-    });
-  }
+  let results;
+  switch(round.type) {
+    case RoundTypes.impact:
+      // NOTE: return type is different. This is a bug.
```
Comment (thelostone-mc) [inline src/server/api/routers/results.ts:141]: @0xKurt

File: src/components/ConnectButton.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ export const ConnectButton = ({ children }: PropsWithChildren) => {
                     variant="primary"
                     data-testid="connect-wallet"
                   >
-                    {isMobile ? "Connect" : "Connect wallet"}
+                    Connect wallet
```
Comment (hussedev) [inline src/components/ConnectButton.tsx:69]: was this to avoid the hydration error?

File: public/metrics.csv
Comment (hussedev) [inline public/metrics.csv:1]: Nice!

File: src/features/applications/components/ApplicationsToApprove.tsx [hunk 3]
```diff
@@ -61,9 +61,11 @@ export function ApplicationItem({
...
         <div className="flex-1">
           <div className="flex items-center justify-between">
-            <Skeleton isLoading={isLoading} className="mb-1 min-h-5 min-w-24">
-              {name}
-            </Skeleton>
+            <Link href={`/${domain}/applications/${id}`} className="hover:underline">
+              <Skeleton isLoading={isLoading} className="mb-1 min-h-5 min-w-24">
+                {name}
+              </Skeleton>
+            </Link>
```
Comment (hussedev) [inline src/features/applications/components/ApplicationsToApprove.tsx:68]: this is fixed in pr #108, just merged, added an eye icon

File: src/features/distribute/components/Distributions.tsx [hunk 4]
```diff
@@ -41,6 +41,8 @@ export function Distributions() {
     );
   }
 
+  console.log("distributionResult", distributionResult);
+
```
Comment (hussedev) [inline src/features/distribute/components/Distributions.tsx:45]: is this console.log intentional?

File: src/features/distribute/components/ExportCSV.tsx
Comment (hussedev) [inline src/features/distribute/components/ExportCSV.tsx:1]: Nice catch with the icons change!!!!!

File: src/features/metrics/components/MetricsSidebar.tsx [hunk 5]
```diff
@@ -142,6 +143,7 @@ export function MetricsSidebar({
         </ScrollArea>
 
         {footer}
+        <SubmitBallotButton />
```
Comment (hussedev) [inline src/features/metrics/components/MetricsSidebar.tsx:146]: MetricsSidebar is also rendered by MetricDetailsSidebar, that is used in `/pages/[domain]/metrics/[metricId]/index.tsx`

is a page that is not related to the ballot,  only shows information about an specific metric and shouldn't have the SubmitBallotButton.

The Button should be in BallotSidebar for now that is rendered only in the Ballot/Metrics pages

**Review by hussedev — CHANGES_REQUESTED** [review]
Left some comments, and required changes:

the changes are regarding the ApplicationsToApprove issue to go to the application page that is solved already and regarding the Submit Ballot Button that it has to go in the BallotsSidebar

File: src/components/ConnectButton.tsx (hunk 2 above)
Comment (thelostone-mc) [reply src/components/ConnectButton.tsx:69]: Yes.

File: src/features/distribute/components/Distributions.tsx (hunk 4 above)
Comment (thelostone-mc) [reply src/features/distribute/components/Distributions.tsx:45]: yes. will cleanup once we make sure entire flow works

**Review by 0xKurt — APPROVED** [review]
